services/core-web/src/components/mine/Tailings/MineTailingsTable.js

```diff
@@ -60,10 +61,12 @@ export class MineTailingsTable extends Component {
         ),
       },
       {
-        title: "Independent Tailings Review Board",
-        dataIndex: "has_itrb",
+        title: "Independent Tailings Review Board Exemption",
+        dataIndex: "itrb_exemption_status_code",
         render: (text) => (
-          <div title="Independent Tailings Review Board">{BOOLEAN_OPTIONS_HASH[text]}</div>
+          <div title="Independent Tailings Review Board Exemption">
+            {this.props.itrmExemptionStatusCodeHash[text] || Strings.EMPTY_FIELD}
```

Comment:
As an outcome of the discussion with Aaron, update the title to be a little more clear to the user "Has Independent Tailings Review Board?"

Reply:
fixed.